lib/matplotlib/style/__init__.py

```diff
@@ -1,3 +1,6 @@
 from __future__ import absolute_import
 
 from .core import use, context, available, library, reload_library
+from matplotlib import rcParams
+if rcParams['style']:
+    use(rcParams['style'])
```

Comment:
That is true, but I was not able to put it anywhere else... I tried to import the style from rc on matplotlib import, but it is not possible to do it from the mpl.__init__ as the style module uses some functions from the __init__.